ui/actions/nuxeo-move-documents-down-button.js

```diff
@@ -55,8 +55,8 @@ import '../nuxeo-button-styles.js';
         <dom-if id="availability" if="[[_available]]">
           <template>
             <div class="action">
-              <paper-icon-button noink icon="icons:arrow-downward"></paper-icon-button>
-              <span class="label" hidden$="[[!showLabel]]">[[_label]]</span>
+              <paper-icon-button noink icon="icons:arrow-downward" aria-labelledby="label"></paper-icon-button>
```

Comment:
@semisse: did we create a ticket to refactor this code, that we keep repeating?

Reply:
here it is: https://jira.nuxeo.com/browse/ELEMENTS-1340